platform/osgi/platform-osgi-conditions/src/main/java/org/codice/ddf/condition/BundleNameCondition.java

```diff
@@ -76,12 +55,12 @@ public boolean isSatisfied() {
       return false;
     }
     String bundleName = bundle.getHeaders().get("Bundle-SymbolicName");
```

Comment:
The constructor and isSatisfied call actually end up happening on the same thread since these objects are created on the fly using reflection, so it yields no performance benefit. That was why I ripped out the REGEX call and moved it back to the bundle activator to change the data that the condition is handed, unfortunately there is no way to know the bundles ahead of time.